examples/swift/README.md

```diff
@@ -0,0 +1,26 @@
+## Swift examples
+
+### Permissions
+
+Before running the examples, you have to add execute permissions.
+
+```
+$ chmod +x count.swift
+$ chmod +x greeter.swift
```

Comment:
You should be able to chmod +x the file and check into git. Anyone who clones will get the right perms.